packages/runtime/src/widgets/Form/Checkbox.tsx

```diff
@@ -37,7 +50,7 @@ export const CheckboxWidget: React.FC<CheckBoxProps> = (props) => {
             : undefined),
         }}
       >
-        {values?.trailingText}
+        {trailingContent}
```

Comment:
Trailing and leading text are mutually exclusive so we can support either or and use direction: LTR/RTL to make it appear before and after